backend/routes/performance_routes.py

```diff
@@ -52,3 +52,19 @@ def update_performance(performance_id):
 def get_adjudications(performance_id):
 	adjudications = Adjudication.get_by(performance_id=performance_id)
 	return jsonify({adjudication.id: adjudication.to_dict() for adjudication in adjudications})
+
+@blueprint.route('/<performance_id>/adjudications', methods=['POST'])
+def create_adjudication(performance_id):
+    adjudication_json = request.get_json()
+    adjudication_json['performance_id'] = performance_id
+    new_adjudication = Adjudication.create(**adjudication_json)
+
+    return jsonify(new_adjudication.to_dict())
+
+@blueprint.route('/<performance_id>/adjudications/<adjudications_id>', methods=['POST'])
+def update_adjudication(performance_id, adjudications_id):
```

Comment:
don't need performance id

Reply:
@imruahmed  do you think it's sufficient to just remove the performance_id from the route or does it make more sense to create an adjudication_routes.py file with this endpoint inside?

Reply:
discussed offline, will move the update endpoint to its own route file but leave the other endpoints in place.